libraries/angular/projects/chat-ui/src/library/chat-ui.component.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, Input} from '@angular/core';
+import { connectApi, loadChatUi } from "chatkitty";
+
+@Component({
+  selector: 'ChatUi',
+  imports: [],
+  template: `
+    <div id="chat-ui"></div>
+  `,
+  standalone: true,
+  styles: ``
+})
+export class ChatUi {
+
+  @Input({ required: true }) apiKey: string = "";
+
+  @Input({ required: true }) username: string = "";
+
+  @Input({ required: true }) widgetId: string = "";
+
+
+  async ngOnInit() {
+
+    const initializeChat = async () => {
```

Comment:
We can inline `initializeChat` here since Angular's `ngOnInit` is an async function.